app/Models/Profile.ts

```diff
@@ -2,32 +2,21 @@ import { DateTime } from 'luxon'
 import { BaseModel, column } from '@ioc:Adonis/Lucid/Orm'
 
 export default class Profile extends BaseModel {
+  
   @column({ isPrimary: true })
   public id: number
 
-  // @column({ columnName: 'user_id', isPrimary: true })
-  // public id: number
-
   @column()
   public name: string
 
-  @column()
-  public email: string
-
-  @column({ serializeAs: null })
-  public password: string
-
   @column()
   public gender: string
 
   @column()
   public mobile: number
 
   @column()
-  public dob: DateTime
-
-  // @column()
-  // public avatarUrl: string | null
+  public date_of_birth: DateTime
```

Comment:
```ts
public dateOfBirth: DateTime
```

In models, we use camelCase for column names and in db, we use snake_case for column names